<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbOpenPath">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select path find</string>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="text">
         <string>Select path</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>path find</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbOnlyFiles">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>No check hash files</string>
        </property>
        <property name="text">
         <string>Only file name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pbStartFind">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Start find</string>
        </property>
        <property name="text">
         <string>Find duplicates</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxExclude">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Exclude mask</string>
        </property>
        <property name="text">
         <string>Exclude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Mask files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="toolTip">
         <string>Mask files, separated by space</string>
        </property>
        <property name="placeholderText">
         <string>Example: *.jpg *.png file.txt</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>File types</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pbDeleteFiles">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Start delete checked files</string>
        </property>
        <property name="text">
         <string>Delete selected files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbRenameFiles">
        <property name="text">
         <string>Rename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMoveFiles">
        <property name="text">
         <string>Move files</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuUser_colors_row">
     <property name="title">
      <string>User colors row</string>
     </property>
     <addaction name="actionSet_color_1"/>
     <addaction name="actionSet_color_2"/>
     <addaction name="actionSet_dels_color"/>
     <addaction name="actionDefaults_color"/>
    </widget>
    <widget class="QMenu" name="menuChange_language">
     <property name="title">
      <string>Change language</string>
     </property>
     <addaction name="actionRu"/>
     <addaction name="actionEn"/>
    </widget>
    <addaction name="menuUser_colors_row"/>
    <addaction name="menuChange_language"/>
    <addaction name="actionSave_duplicate_list"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_program"/>
    <addaction name="actionChangelog"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSet_color_1">
   <property name="text">
    <string>Set color 1</string>
   </property>
  </action>
  <action name="actionSet_color_2">
   <property name="text">
    <string>Set color 2</string>
   </property>
  </action>
  <action name="actionSet_dels_color">
   <property name="text">
    <string>Set dels color</string>
   </property>
  </action>
  <action name="actionSave_duplicate_list">
   <property name="text">
    <string>Save duplicate list</string>
   </property>
  </action>
  <action name="actionDefaults_color">
   <property name="text">
    <string>Defaults color</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_program">
   <property name="text">
    <string>About program</string>
   </property>
  </action>
  <action name="actionChangelog">
   <property name="text">
    <string>Changelog</string>
   </property>
  </action>
  <action name="actionRu">
   <property name="text">
    <string>Ru</string>
   </property>
  </action>
  <action name="actionEn">
   <property name="text">
    <string>En</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
